### DISCUSSION TITLE
EME on wordpress.org

### DISCUSSION THREAD

**Discussion by chrkretz** [liedekef/events-made-easy#577] [General]
Hi,
I have been living under the impression that the EME plug-in along with its updates is available on wordpress.org. Now I got a warning from my security plug-in that this is no longer the case and realizing that I am still on version 2.3.13 so assuming I have missed something...
Thanks for any insights.

**Comment by liedekef** [comment]
Read the reason and how to upgrade here: https://www.e-dynamics.be/wordpress/forums/topic/security-release/

**Reply by chrkretz** [reply]
Thanks for the quick response and much appreciated!
As a matter of fact I was hoping for this best-case scenario meaning EME updates should be available on wordpress.org again sooner or later - sorry for the trouble you are going through to get it back on the platform.
While we had been using EME for online reservations (w/o online payment) for our non-profit Jazz events for quite a while, I have only recently added Mollie as a payment platform and must say the plug-in works like a charm. Thanks for all the work you are investing in EME - I am aware that kudos and acknowledgements do not pay anyone's meals etc. so our donation is forthcoming...

**Reply by liedekef** [reply]
Thanks! Make sure to update to the latest EME version on github. After that, the automatic updates will work for you as usual (albeit not via wordpress.org, but you won't notice that)

**Reply by chrkretz** [reply]
Unfortunately the update to version 2.4.47 failed: I deleted the /wp-content/plugins/events-made-easy folder and uploaded the new folder/files via FTP, which resulted in a "fatal error". By replacing the new with the old files I was able to restore the old version (2.3.13) and it looks like all data are still there.
Any help with this would be appreciated.
Thanks,
Christian

**Reply by liedekef** [reply]
A fatal error? That probably means you're still on php 7, which is end of life since the beginning of last year ... change your php to at least 8.1 or higher (see https://www.php.net/supported-versions.php )

**Reply by chrkretz** [reply]
Right, server was still running on PHP 7.4. I switched to 8.3 now and the update went smoothly. Everything seems to work fine from what I can see now. Also I understand that any future updates via github can be triggered from the Wordpress backend (or automatically) as usual, right?
Thanks very much for the quick support, which is much appreciated.
Cheers,
Christian

**Reply by liedekef** [reply]
indeed, future updates happen as usual.

**Reply by chrkretz** [reply]
Thanks again!
